dotnet/simple-app/Program.cs

```diff
@@ -0,0 +1,284 @@
+            Console.WriteLine(e.ToString());
+        }
+        await client.Close();
+    }
+    
+    private static void SyncOpenConnectionExample()
+    {
+        var client = ImmuClientSync.NewBuilder().WithServerUrl(immudbServerAddress).Build();
+        client.Open("immudb", "immudb", "defaultdb");
+
+        string key = "hello";
+
+        try
+        {
+            client.VerifiedSet(key, "immutable world!");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+        client.Close();
+    }
+
+    private static async Task AnotherOpenConnectionExample()
+    {
+        var client = await ImmuClient.NewBuilder().WithServerUrl(immudbServerAddress).Open();
+        string key = "hello";
+
+        try
+        {
+            // Setting (adding) a key-value.
+            await client.VerifiedSet(key, "immutable world!");
+
+            // Getting it back, by key (in a verified way that reports any tampering if it happened).
+            Entry entry = await client.VerifiedGet(key);
+            Console.WriteLine($"{key}, {entry.ToString()}");
+        }
+        catch (VerificationException e)
+        {
+            // VerificationException means Data Tampering detected!
+            // This means the history of changes has been tampered.
+            Console.WriteLine(e.ToString());
+        }
+
+        await client.Close();
+    }
+
+    private static async Task SetAllGetAllExample()
+    {
+        var client = new ImmuClient(immudbServerAddress, 3322);
+        await client.Open("immudb", "immudb", "defaultdb");
+
+
+        try
+        {
+            List<string> keys = new List<string>();
+            keys.Add("k0");
+            keys.Add("k1");
+
+            List<byte[]> values = new List<byte[]>();
+            values.Add(new byte[] { 0, 1, 0, 1 });
+            values.Add(new byte[] { 1, 0, 1, 0 });
+
+            List<KVPair> kvListBuilder = new List<KVPair>();
+
+            for (int i = 0; i < keys.Count; i++)
+            {
+                kvListBuilder.Add(new KVPair(keys[i], values[i]));
+            }
+            await client.SetAll(kvListBuilder);
+
+            List<Entry> getAllResult = await client.GetAll(keys);
+
+            for (int i = 0; i < getAllResult.Count; i++)
+            {
+                Entry entry = getAllResult[i];
+                Console.WriteLine($"({string.Join(" ", entry.Key)}, {keys[i]}):({string.Join(" ", entry.Value)}, {string.Join(" ", values[i])})");
+            }
+
+            await client.Close();

```

Comment:
This is repeated after the `catch` clause

Reply:
fixed